CMakeLists.txt

```diff
@@ -1,12 +1,23 @@
 cmake_minimum_required(VERSION 3.16.0)
 
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
+
 project(swift-argument-parser
   LANGUAGES Swift)
 
 option(BUILD_EXAMPLES "Build Example Programs" TRUE)
+option(BUILD_SHARED_LIBS "Build shared libraries by default" YES)
 
 include(CTest)
 
+if(CMAKE_VERSION VERSION_LESS 3.16 AND CMAKE_SYSTEM_NAME STREQUAL Windows)
```

Comment:
Is this needed?  The minimum for this project is currently 3.16.  I think that we can ignore the <3.16 compatibility

Reply:
you're right, this was unnecessary, I dropped it